app/components/Upload/UploadImage.css

```diff
@@ -48,7 +54,7 @@
 }
 
 .placeholdeTitle {
-  background: rgba(255, 255, 255, 0.8);
+  background: var(--color-white);
```

Comment:
Make sure this doesn't change anything

Reply:
Moved to the original color, using rgba-variables